server/routes/viewRoutes.test.ts

```diff
@@ -371,6 +390,27 @@ afterEach(() => {
 })
 
 describe('View journey routes tests', () => {
+  describe('Get nomis calculation summary view tests', () => {
+    it('GET /view/:nomsId/nomis-calculation-summary/:offenderSentCalculationId should have the correct details', () => {
+      prisonerService.getPrisonerDetail.mockResolvedValue(stubbedPrisonerData)
+      calculateReleaseDatesService.getNomisCalculationSummary.mockResolvedValue(pastNomisCalculation as never)
+      return request(app)
+        .get('/view/A1234AA/nomis-calculation-summary/-1')
+        .expect(200)
+        .expect(res => {
+          expect(res.text).toContain('A1234AA')
```

Comment:
Could you use cheerio for these fields and use strict equals? You can add some data-qa in your markup to make it easy

Reply:
Amended as requested.